### ISSUE TITLE
packager: Automatically push family updates

### ISSUE THREAD

**Issue by m4rc1e** [googlefonts/gftools#285]
Since google/fonts now uses Github Actions, we could make an action to run the packager on each upstream family file so the latest fonts are auto pushed on a specific day.

It appears that GA has a feature similar to [cron](https://jasonet.co/posts/scheduled-actions/) so it does seem very feasible.

@graphicore Does the packager run a checksum diff before submitting a PR?

**Comment by graphicore** [comment]
> @graphicore Does the packager run a checksum diff before submitting a PR?

No, but it uses git, so we should be able to easily check if there are any changes in the branch compared to master.

**Comment by graphicore** [comment]
> on each upstream family file so the latest fonts are auto pushed on a specific day

If we run updates on the upstream master branch HEAD we may just push a lot of work in progress. There's no rule that says the contents of the upstream master branch must always be in a publishable state, A better solution would be e.g. to configure the `upstream.yaml` to push from a specific tag instead of a branch, e.g. `latest`. But that would require the upstream maintainer to maintain that `latest` tag, and hence it wouldn't be fully automated any more.

**Comment by graphicore** [comment]
Using tags instead of branches is already possible with the packager.
